com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/flowable/ResumeProcessAction.java

```diff
@@ -39,8 +40,9 @@ public void executeActualProcessAction(String userId, String superProcessInstanc
     private void triggerProcessInstance(String userId, String processId) {
         List<Execution> executionsAtReceiveTask = flowableFacade.findExecutionsAtReceiveTask(processId);
         if (!executionsAtReceiveTask.isEmpty()) {
-            executionsAtReceiveTask.stream()
-                .forEach(execution -> flowableFacade.trigger(userId, execution.getId()));
+            for (Execution execution : executionsAtReceiveTask) {
+                flowableFacade.trigger(userId, execution.getId());
```

Comment:
Nice! Don't show this to Petko, though. :D

Reply:
![image](https://user-images.githubusercontent.com/43955302/60726454-3eb2ab80-9f44-11e9-9fbf-47fc2862bcb8.png)

Reply:
Venkat is turning in his bed...

Reply:
On the plus side, no one from the team will curse you when looking at a stack trace coming from this loop. Can't say the same about Petko, though... @petko-boyadzhiev If some day your ears are burning, try to remember whether you've used .forEach() that day. :P

Reply:
The north will remember. The winter streams are going to pluck you loops from the ground and blow them to the past...

Reply:
https://www.youtube.com/watch?v=UY3oZH6Piz4&feature=youtu.be&t=126